combat-trainer.lic

```diff
@@ -246,10 +249,29 @@ class LootProcess
     game_state.mob_died = true
     arrange_mob(DRRoom.dead_npcs.first, game_state)
     if @necro
-      if @ritual_type == 'harvest'
+      if @heal
+        wounds = check_wounds
+        echo "Wounds: #{wounds}" if $debug_mode_ct
+        unless wounds.empty?
+          Flags.add('necro-spellcast', '^You feel fully prepared to cast your spell\.')
+          prev_ritual = @ritual_type
```

Comment:
I'm not a fan of this logic, but I'd have to grok the code better to articulate why. Can the consume code be commonized and called from here instead of changing @ritual_type?